### PR TITLE
Should not be using decimal math here as it is way slower than floating point

### PR DESCRIPTION
**Pull request by kendallb**
I mentioned this in this issue:

https://github.com/ClosedXML/ClosedXML/issues/545

Pretty simple fix but performance will suffer using decimal math here. All unit tests pass but as far as I can tell there is not actual test for this specific function.

- [ ] C# Code Review: @csreviewer
- [ ] Test Automation Review: @csreviewer

### CODE REVIEW

File: ClosedXML_Sandbox/PerformanceRunner.cs [hunk 1]
```diff
@@ -40,6 +40,23 @@ public static void RunInsertTable()
             EmulateSave(workbook);
         }
 
+        public static void TimeDecimalRound()
```
Comment (igitur) [inline ClosedXML_Sandbox/PerformanceRunner.cs]: Undo the changes in this file, please.

**Comment by kendallb** [comment]
I removed the performance runner changes

**Comment by igitur** [comment]
Thanks. This is scheduled for v0.91.
